lesson2/src/lesson2.java

```diff
@@ -0,0 +1,53 @@
+package lesson2;
+
+public class lesson2 {
+    public static void main(String[] args) {
+
+        //Проверка работы методов
+        PolSumPrint(-1);
+        MultiPrint("Hello World!", 5);
+        System.out.println("NeedSum " + NeedSum(7, 3));
+        System.out.println("OtrSumBool " + OtrSumBool(0));
+        int year = 195;
+        if (Visocos(year))
+            System.out.println("Год " + year + " високосный");
+        else
+            System.out.println("Год " + year + " не високосный");
+        //конец проверки методов
+    }
+
+    public static boolean NeedSum(int a, int b) {
+        if (a + b >= 10 && a + b <= 20)
```

Comment:
поскольку логическое выражение само по себе возвращает boolean результат, то можно упростить:
```suggestion
        return a + b >= 10 && a + b <= 20;
```